image/base/locale.gen

```diff
@@ -0,0 +1,506 @@
+# This file lists locales that you wish to have built. You can find a list
+# of valid supported locales at /usr/share/i18n/SUPPORTED, and you can add
+# user defined locales to /usr/local/share/i18n/SUPPORTED. If you change
+# this file, you need to rerun locale-gen.
+
+
+C.UTF-8 UTF-8
```

Comment:
Is there a reason we need to generate this?

Reply:
We could probably remove this too